services/manuals-frontend/docker-compose.yml

```diff
@@ -0,0 +1,57 @@
+version: '3.7'
+
+x-manuals-frontend: &manuals-frontend
+  build:
+    context: .
+    dockerfile: services/manuals-frontend/Dockerfile
+  image: manuals-frontend
+  volumes:
+    - ..:/govuk:delegated
```

Comment:
This is now `$GOVUK_ROOT_DIR:/govuk:delegated`

Reply:
Thanks @barrucadu, just as I thought it was safe to go into the repo...

Reply:
Never underestimate my desire to suddenly change everything